### PR TITLE
Show products homepage

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- [#45542](https://edu-redmine.sun-asterisk.vn/issues/45542)

## WHAT (optional)
- Show newest products in homepage

## HOW
- fake data via faker gem
- Render products

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: app/assets/stylesheets/style.css [hunk 1]
```diff
@@ -678,16 +678,20 @@ p.m_text {
 	color: #00405d;
 }
 .grid-product{
-	display:block;
+	display: flex;
```
Comment (huongnt-2545) [inline app/assets/stylesheets/style.css]: check lại indent trong file này em nhé
để indent = 2 spaces

File: app/models/book.rb [hunk 2]
```diff
@@ -7,4 +7,6 @@ class Book < ApplicationRecord
   has_many :authors, through: :book_authors
   has_many :comments, dependent: :destroy
   has_many :rates, dependent: :destroy
+
+  scope :newest, ->{order(created_at: :desc)}
```
Comment (huongnt-2545) [inline app/models/book.rb]:
```suggestion
  scope :newest, ->{ order created_at: :desc }
```

File: app/views/books/_book.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div class="  product-grid">
+  <div class="content_box">
+    <a href="single.html">
+      <div class="left-grid-view grid-view-left">
+        <%= link_to image_tag("pic1.jpg", alt: "product", class: "img-responsive watch-right"), "#" %>
+        <div class="mask">
+          <div class="info">Quick View</div>
```
Comment (huongnt-2545) [inline app/views/books/_book.html.erb]: I18n

File: app/views/books/_book.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<div class="  product-grid">
```
Comment (huongnt-2545) [inline app/views/books/_book.html.erb]:
```suggestion
<div class="product-grid">
```

File: app/views/books/_book.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    <a href="single.html">
+      <div class="left-grid-view grid-view-left">
+        <%= link_to image_tag("pic1.jpg", alt: "product", class: "img-responsive watch-right"), "#" %>
+        <div class="mask">
+          <div class="info">Quick View</div>
+        </div>
+      </div>
+    </a>
+      <h4><%= link_to book.name %></h4>
+      <span>$ <%= link_to book.price %></span>
```
Comment (huongnt-2545) [inline app/views/books/_book.html.erb]: dùng I18n currency để format tiền tệ nhé

File: app/views/home_pages/home.html.erb [hunk 6]
```diff
@@ -14,21 +14,8 @@
...
-				</a>
-					</div>
-					<h4><a href="#">Ra Bờ Suối Ngắm Hoa Kèn Hồng </a></h4>
-					130.500 đ
-			</div>
-		</div>
-
-			<div class="clearfix"> </div>
-		</div>
+		<%= render @books %>
```
Comment (huongnt-2545) [inline app/views/home_pages/home.html.erb]: check indent file này

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**